<launch>
  <arg name="save_dir" default="$(env HOME)/deviation_evaluator_sample"/>
  <arg name="localization_error_monitor_param_path" default="$(find-pkg-share tier4_localization_launch)/config/localization_error_monitor.param.yaml"/>

  <node pkg="deviation_evaluator" exec="deviation_evaluation_visualizer.py" name="deviation_evaluation_visualizer" output="screen">
    <param name="save_dir" value="$(var save_dir)"/>
    <param from="$(var localization_error_monitor_param_path)"/>
  </node>
</launch>
